lang/cs/Org.Apache.REEF.ClrDriver/AssemblyInfo.cpp

```diff
@@ -48,8 +48,8 @@ using namespace System::Security::Permissions;
 // You can specify all the value or you can default the Revision and Build Numbers
 // by using the '*' as shown below:
 
-[assembly:AssemblyVersion("0.13.0.0")]
-[assembly:AssemblyFileVersion("0.13.0.0")]
```

Comment:
This file should also be included in change_version.py, so that the version in it gets updated together with others.